otx/algorithms/classification/tasks/__init__.py

```diff
@@ -26,7 +26,7 @@
 
 __all__ = [
     "MPAClassification",
-    "MPABackbones",
+    "OTXBackbones",
```

Comment:
I can't find using `MPABackbones`. Should `OTXBackbones` still need to be in `__all__`?

Reply:
the role of registering the backbone in the registry by importing it. These are required imports that are not used directly in our code, but only serve to register in the registry.

Reply:
but does it need to be in __all__, not just import it in __init__.py?

Reply:
I put it in with the thought that it might not be used someday.. Is it neat to erase it?

Reply:
it won't make problem. I just think that useless functions don't have to be imported when this file is imported.
but if you think they'll be used later, then I can understand, of course.